doc/SLSKPROTOCOL.md

```diff
@@ -231,64 +235,53 @@ and callbacks for the messages are set in pynicotine.py.
-| ----------- | -------------- | ------------ | --------------- | ----------------------- | --------------- | ----------------------- |
-| Human       | 72             | 1            | 8               | username                | 8               | password                |
-| Hex         | 48 00 00 00    | 01 00 00 00  | 08 00 00 00     | 75 73 65 72 6e 61 6d 65 | 08 00 00 00     | 70 61 73 73 77 6f 72 64 |
+#### Sending Login Example
 
-*Message, continued*
+*Message:*
+| *Data*    | Message Length | Message Code | Username Length | Username                | Password Length | Password                |
+| --------- | -------------- | ------------ | --------------- | ----------------------- | --------------- | ----------------------- |
+| **Human** | 72             | 1            | 8               | username                | 8               | password                |
+| **Hex**   | 48 00 00 00    | 01 00 00 00  | 08 00 00 00     | 75 73 65 72 6e 61 6d 65 | 08 00 00 00     | 70 61 73 73 77 6f 72 64 |
 
-| Description | Version     | Length      | Hash                                                                                            | Minor Version |
-| ----------- | ----------- | ----------- | ----------------------------------------------------------------------------------------------- | ------------- |
-| Human       | 160         | 32          | d51c9a7e9353746a6020f9602d452929                                                                | 1             |
-| Hex         | a0 00 00 00 | 20 00 00 00 | 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36 30 32 64 34 35 32 39 32 39 | 01 00 00 00   |
+*... continued:*
+| *Data*    | Version     | Length      | Hash                                                                                            | Minor Version |
+| --------- | ----------- | ----------- | ----------------------------------------------------------------------------------------------- | ------------- |
+| **Human** | 160         | 32          | d51c9a7e9353746a6020f9602d452929                                                                | 1             |
+| **Hex**   | a0 00 00 00 | 20 00 00 00 | 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36 30 32 64 34 35 32 39 32 39 | 01 00 00 00   |
 
-*Message as a Hex Stream* **48 00 00 00 01 00 00 00 08 00 00 00 75 73 65
-72 6e 61 6d 65 08 00 00 00 70 61 73 73 77 6f 72 64 a0 00 00 00 20 00 00
-00 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36
-30 32 64 34 35 32 39 32 39 01 00 00 00**
+*Message as a Hex Stream:*
+  - **48 00 00 00 01 00 00 00 08 00 00 00 75 73 65 72 6e 61 6d 65 08 00 00 00 70 61 73 73 77 6f 72 64 a0 00 00 00 20 00
+      00 00 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36 30 32 64 34 35 32 39 32 39 01 00 00 00**
 
-### Data Order
-
-  - Send
+*Data Order:*
```

Comment:
Visually and semantically, I still prefer a heading here. Please revert to a level 3 heading, as these are all sections under `Server Code 1`.

Reply:
Perhaps, the "Data Order" headings are not needed at all?